src/main/java/org/ice4j/stack/Connector.java

```diff
@@ -207,6 +207,13 @@ else if(localSock.getUDPSocket() != null)
                     return;
 
                 logger.finest("received datagram");
+                logger.finest(String.format("packet - addr: %s port: %d",
+                    packet.getAddress(), packet.getPort()));
```

Comment:
Could you please combine these two log messages into one? In a multithreaded environment they could otherwise end up being separated in the log.